routes/api.php

```diff
@@ -14,6 +17,15 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
-});
+Route::get('/movies', [MoviesController::class, 'list']);
+Route::post('/movies/create', [MoviesController::class, 'create']);
+Route::get('/movies/{movie}', [MoviesController::class, 'show']);
+Route::put('/movies/{movie}/edit', [MoviesController::class, 'update']);
+Route::delete('/movies/{movie}/delete', [MoviesController::class, 'delete']);
```

Comment:
по REST не нужны суффиксы, url выступает как идентификатор ресурса над которым мы делаем операцию
http метод указывает операцию которую мы хотим совершить с ресурсом
- get - получить
- post - создать
- put - редактировать
- delete - удалить 
```suggestion
Route::put('/movies/{movie}', [MoviesController::class, 'update']);
Route::delete('/movies/{movie}', [MoviesController::class, 'delete']);
```